activesupport/lib/active_support/test_case.rb

```diff
@@ -134,20 +134,147 @@ def parallelize_teardown(&block)
     include ActiveSupport::Testing::FileFixtures
     extend ActiveSupport::Testing::Declarative
 
-    # test/unit backwards compatibility methods
-    alias :assert_raise :assert_raises
+    ##
+    # :method: assert_not_empty
+    #
+    # :call-seq:
+    #   assert_not_empty(obj, msg = nil)
+    #
+    # Alias for: refute_empty[http://docs.seattlerb.org/minitest/Minitest/Assertions.html#method-i-refute_empty]
+
+    #
```

Comment:
If there's an rdoc wizard who can figure out how to make this work without this extra `#`, that would be awesome. When I try removing it the `:method:` above will no longer show up

Reply:
My best guess, the extra comment is only necessary because RDoc is likely throwing away the comment otherwise due to not being able to complete the connection between the alias. This is because the target method does not exist in this context.

If we dynamically create the method using the same method above, and move the comment above the alias, RDoc will then create a reference to both methods and include the "also aliased as" link in the generated docs.

Since that way requires duplicating documentation of the source method from minitest, I think this is our best option to just create a space between the magic method doc and the alias.

Another alternative is to put the magic comments together and separate from the code, but this way they are relatively close to their implementation.

Sorry that's not the best answer, but based on my experiments this seems OK.